crates/bevy_ecs/src/reflect/from_world.rs

```diff
@@ -0,0 +1,84 @@
+//! Definitions for [`FromWorld`] reflection.
+//!
+//! This module exports two types: [`ReflectFromWorldFns`] and [`ReflectFromWorld`].
+//!
+//! Same as [`super::component`], but for [`FromWorld`].
+
+use bevy_reflect::{FromType, Reflect};
+
+use crate::world::{FromWorld, World};
+
+/// A struct used to operate on reflected [`FromWorld`] of a type.
```

Comment:
This line isn't clear to me. It feels like we're missing a noun after `FromWorld`.

Reply:
Yes please :)